### ISSUE TITLE
[Feature Request]: Autogen Studio - Disable code execution

### ISSUE THREAD

**Issue by Yanni8** [microsoft/autogen#2031]
### Is your feature request related to a problem? Please describe.

Maybe this feature already exists, but I wasn't able to find a method to disable the code execution in the Autogen Studio UI.
And as far as I know does autogen studio currently not use Docker to execute the code isolated. 

### Describe the solution you'd like

I would recommend two different _functionalities_

1. Autogestudio should use Docker by default to execute code
2. It should be possible to disable code execution completely

**Comment by victordibia** [comment]
@Yanni8 ,

Yes, you are right. The UI does not surface the code execution configuration right now. 
Will provide an update here once this is added. 

In the mean time, if you would like to contribute to this, let me know.

**Comment by Yanni8** [comment]
Hey @victordibia,

I could try to implement it. But I first have to familiarize myself with the Autogenstudio project structure.

**Comment by rysweet** [comment]
removing 0.2 label.

**Comment by jackgerrits** [comment]
Code execution should be configurable in 0.4 studio now
